docs/guides/kubernetes/deploy-tobs-on-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,174 @@
+lke-monitor-replica                                         ClusterIP   10.128.137.189   <none>        5432/TCP                     4m3s
+opentelemetry-operator-controller-manager-metrics-service   ClusterIP   10.128.45.42     <none>        8443/TCP                     4m3s
+opentelemetry-operator-webhook-service                      ClusterIP   10.128.12.89     <none>        443/TCP                      4m3s
+prometheus-operated                                         ClusterIP   None             <none>        9090/TCP                     3m41s
+tobs-kube-prometheus-alertmanager                           ClusterIP   10.128.33.44     <none>        9093/TCP                     4m3s
+tobs-kube-prometheus-operator                               ClusterIP   10.128.175.39    <none>        443/TCP                      4m3s
+tobs-kube-prometheus-prometheus                             ClusterIP   10.128.106.173   <none>        9090/TCP                     4m3s
+
+    {{< /output >}}
+
+    From the above output, the resource services you will access have the corresponding ports:
+
+    | Resource     | Service Name                      | Port |
+    | ------------ | ----------------------------------| ---- |
+    | Prometheus   | tobs-kube-prometheus-prometheus   | 9090 |
+    | Alertmanager | tobs-kube-prometheus-alertmanager | 9093 |
+    | Grafana      | lke-monitor-grafana               | 80   |
+
+1.  Use `kubectl` [port-forward](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#port-forward) to open a connection to a service, then access the service's interface by entering the corresponding address in your web browser:
+
+    {{< note >}}
+Press **control+C** on your keyboard to terminate a port-forward process after entering any of the following commands.
+    {{< /note >}}
+
+    *   To provide access to the **Prometheus** interface at the address `127.0.0.1:9090` in your web browser, enter:
+
+            kubectl -n monitoring \
+            port-forward \
+            svc/tobs-kube-prometheus-prometheus \
+            9090
+
+    *   To provide access to the **Alertmanager** interface at the address `127.0.0.1:9093` in your web browser, enter:
+
+            kubectl -n monitoring \
+            port-forward \
+            svc/tobs-kube-prometheus-alertmanager  \
+            9093
+
+    *   To provide access to the **Grafana** interface at the address `127.0.0.1:8081` in your web browser, enter:
+
+            kubectl -n monitoring \
+            port-forward \
+            svc/lke-monitor-grafana  \
+            8081:80
+
+
+When accessing the Grafana interface, log in as `admin`. You can get the `password` using: `kubectl get secret --namespace monitoring lke-monitor-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo`. The Grafana dashboards are accessible at **Dashboards > Manage** from the left navigation bar.
```

Comment:
Since `kubectl get secret --namespace monitoring lke-monitor-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo` is so long it's probably best to display it on it's own line as a command block.